### PR TITLE
style: Allow scrolling for long names

### PR DESCRIPTION
**Pull request by Akahadaka**
Make the content horizontally scrollable.
**NOTE**: Not tested in other browsers - may have ugly scrollbars
Screen-Recording-2020-06-16-at-1

### CODE REVIEW

File: src/components/Steam.vue [hunk 1]
```diff
@@ -122,13 +124,23 @@ export default Vue.extend({
...
+  overflow-x: scroll;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+  &::-webkit-scrollbar {
+    height: 0;
+  }
+  &-text {
+    white-space: nowrap;
+    padding-right: 10px;
+  }
```
Comment (himohimo) [inline src/components/Steam.vue:138]: Is this duplicating the same functionality from the other component?
At what point is it worth it to add CSS directives into a global stylesheet?

File: src/components/Steam.vue [hunk 2]
```diff
@@ -122,13 +124,23 @@ export default Vue.extend({
...
 }
 .fixed {
   position: fixed;
   // TODO Find out how to contain a position:fixed element in parent bounds
-  width: calc(100vw - 58%);
+  width: calc(50% - 24px);
```
Comment (himohimo) [inline src/components/Steam.vue:143]: Out of curiosity, is there anything wrong with using `rem` everywhere (the current way I'm doing it)?

**Review by himohimo — APPROVED** [review]
Works well 👍 
I wonder how we can make it obvious that one can scroll horizontally

File: src/components/Steam.vue (hunk 1 above)
Comment (Akahadaka) [reply src/components/Steam.vue:138]: I wonder about this too. I do like that the component is "contained". I know it feels like repeating and it'll only get worse if we add an Epic, Origin, Gog component, etc. Perhaps there needs to be a "parent" component that they all extend? Is there a pattern for that in Vue? I imagine that CSS scope will be the main issue.

File: src/components/Steam.vue (hunk 2 above)
Comment (Akahadaka) [reply src/components/Steam.vue:143]: I should probably start using `rem`. I just haven't been and don't have a good intuition of how it scales for margins and padding. In this case the 24px is 2x the default `v-container` padding

**Comment by Akahadaka** [comment]
> Works well 👍
> I wonder how we can make it obvious that one can scroll horizontally

I actually added a `cursor:ew-resize` on the title, but the trick was deciding which ones were overflowing. The best I could come up with was adding a `ref` to the card and looking at each one for where `v-spacer.width == 0` and adding a class to the card as an indicator. This worked quite nicely, except for it being pretty convoluted and having to be duplicated in both components.

**Pull request merged**